src/psd_tools/api/psd_image.py

```diff
@@ -593,6 +593,19 @@ def rec_helper(layer):
 
         rec_helper(self)
 
+    def set_log_level(self, log_level):
+        """
+        Set the logging level.
+
+        :param log_level: string of the desired logging level 'CRITICAL', 'ERROR', 'WARNING', 'INFO' or 'DEBUG'.
+
+        Default logging level is WARNING.
+        """
+        if log_level in ("CRITICAL", "ERROR", "WARNING", "INFO", "DEBUG"):
+            logging.basicConfig(level=log_level)
```

Comment:
There is actually a caveat in resetting the logging configuration once set:
https://stackoverflow.com/questions/12158048/changing-loggings-basicconfig-which-is-already-set

Maybe adding `force=True` option is fine, but that would destroy all the other handlers set outside of the psd-tools. https://docs.python.org/3.8/library/logging.html?highlight=logging#logging.basicConfig

Reply:
You are right. 
In the latest commit I added a logger at the "psd_tools.api" level which serves as the local root of psd_tools when used as a module. 
The new set_log_level() method only impacts the log level of "psd_tools.api" and its children, which is effectively every logger declared inside src/psd_tools/api but nothing outside of it.

Reply:
The commit message should read "Better way to set the log level without impacting **loggers** outside of psd-tools" (not handlers)